plugin/ale/sign.vim

```diff
@@ -40,6 +40,34 @@ execute 'sign define ALEWarningSign text=' . g:ale_sign_warning
 \   . ' texthl=ALEWarningSign'
 sign define ALEDummySign
 
+function! ale#sign#GetStatuslineStr()
```

Comment:
Could you move this into a `statusline.vim` file? I think this belongs in its own file. The `Str` in the name is redundant. I wonder if the "public" functions for ALE should all be named in PascalCase without the paths. Something like `ALEGetStatusLine`.